src/main/scala/esmeta/phase/RangeTypeCheck.scala

```diff
@@ -0,0 +1,74 @@
+package esmeta.phase
+
+import esmeta.*
+import esmeta.cfg.CFG
+import esmeta.analyzer.*
+import esmeta.analyzer.domain
+import esmeta.util.*
+import esmeta.util.SystemUtils.*
+import esmeta.spec.Spec.*
+import scala.io.StdIn
+import esmeta.error.TypeCheckFail
+import esmeta.error.ESMetaError
+
+case object RangeTypeCheck extends Phase[Unit, Unit] {
+  val name = "range-tycheck"
+
+  val help =
+    "performs a type analysis of ECMA-262 through multiple versions of specification."
+
+  def apply(
+    unit: Unit,
+    cmdConfig: CommandConfig,
+    config: Config,
+  ): Unit =
+    val range =
+      config.range.getOrElse(throw new Exception("range is not specified."))
```

Comment:
If you always need range then I think using [cmdConfig.targets](https://github.com/es-meta/esmeta/blob/108268a03d37c0f70010c49d5a80a756073d724a/src/main/scala/esmeta/phase/Test262Test.scala#L36) and  [targetName](https://github.com/es-meta/esmeta/blob/8e6f0c43da171722b70cc0622d72020316167351/src/main/scala/esmeta/Command.scala#L166) seems to be a better choice.

Reply:
I've changed it to use cmdConfig.targets, and now it gets multiple targets the same way git rev-list does. thanks :)